apps/deploy-web/src/instrumentation.ts

```diff
@@ -0,0 +1,13 @@
+export async function register() {
```

Comment:
this is how we can run validation once on server start up

Reply:
ref: https://nextjs.org/docs/app/building-your-application/optimizing/instrumentation